src/openrct2/ride/TrackDesign.cpp

```diff
@@ -911,15 +910,6 @@ static void TrackDesignMirrorMaze(TrackDesign* td6)
     {
         maze.y = -maze.y;
 
```

Comment:
Shouldn't this have been deleted along with the lines below?

Reply:
No, this is correct - if you’re mirroring a maze or ride, you flip it along the y axis.

Reply:
But wasn it already moved to `TrackDesignMirrorEntrances`?

Reply:
The parts that have to do with entrances and exits have indeed been moved to `TrackDesignMirrorEntrances` - the remaining code is for mirroring the actual maze parts. The old code handled both actual maze parts _and_ entrances.

Reply:
I see, thanks!